tests/test_minimal/test_tools.py

```diff
@@ -186,7 +186,7 @@ def setUp(self):
         self.nwb_folder_path = self.tmpdir / "test_nwb"
         self.nwb_folder_path.mkdir()
         metadata = get_default_nwbfile_metadata()
-        metadata["NWBFile"].update(session_start_time=datetime.now().astimezone())
+        metadata["NWBFile"].update(session_start_time=datetime.now().astimezone(), session_id="test_automatic_upload")
```

Comment:
Should we add `neuroconv` to the session_id just for a little bit of better tracking?

Then I saw that the dandi set is called `NeuroConv SDK Testing` so probably not need. Maybe a timestamp could be useful?

Reply:
A timestamp would be dangerous - it would result in complete spam of the dandiset (a new file every time the CI runs)